lib/ddtrace/contrib/configurable.rb

```diff
@@ -30,12 +30,13 @@ def configurations
           end
         end
 
-        def configure(key, options = {}, &block)
-          key = resolver.resolve(key || :default)
+        def configure(key = :default, options = {}, &block)
+          resolver.add_key(key)
```

Comment:
This is how I ended up solving the awkward dance that `RegexpResolver` was doing to remember patterns.